rewards/rewards_checker.py

```diff
@@ -75,24 +76,21 @@ def verify_rewards(past_tree, new_tree, tree_manager: TreeManager, chain: str):
             chain,
         )
         raise e
-
+    table = []
     for name, token in TOKENS_TO_CHECK[chain].items():
         total_before_token = int(past_tree["tokenTotals"].get(token, 0))
         total_after_token = int(new_tree["tokenTotals"].get(token, 0))
+        decimals = 18
```

Comment:
Rather than hardcoding decimals here, we can get the information stored in our config file this way:
```
        token_info = fetch_token(chain, name)
        decimals = token_info.get("decimals", 18)
```
Or
```
    decimals = fetch_token_decimals(chain, name)
```

The latter will return a default of 0, former will return default of 18 through the `.get` on token_info